src/components/views/elements/AppTile.js

```diff
@@ -207,6 +219,14 @@ export default React.createClass({
                     ></iframe>
                 </div>
             );
+        } else if (this.isMixedContent() == true) {
```

Comment:
You can drop the `== true` and also does this work if `hasPermissionToLoad=true` but it's mixed content? 0:)